server/views/partials/footer.njk

```diff
@@ -1,12 +1,6 @@
 {% from "govuk/components/footer/macro.njk" import govukFooter %}
 
-<div style="background: {{ feedbackBannerBackground or '#fff' }}" class="govuk-!-display-none-print">
```

Comment:
this has now been moved into the `main` content as the footer is served by the frontend components